crates/hotshot/Cargo.toml

```diff
@@ -31,6 +31,7 @@ async-lock = { workspace = true }
 async-trait = { workspace = true }
 bimap = "0.6"
 bincode = { workspace = true }
+chrono = "0.4"
```

Comment:
Chrono is imported in `examples` and `task-impls`. To avoid version conflicts, could we perhaps just make the version standard across the workspace? Looks like they're all using 0.4.